packages/react-icons-mdl2/src/utils/createSvgIcon.ts

```diff
@@ -9,12 +9,14 @@ const createSvgIcon = <TProps = {}>({ svg, displayName }: SvgIconCreateFnParams<
     const { className, style = {} } = props;
 
     const nativeProps = getNativeProps<React.HTMLAttributes<HTMLElement>>(props, htmlElementProperties);
-    const containerProps = props['aria-label']
-      ? {}
-      : {
-          role: 'presentation',
-          ['aria-hidden']: true,
-        };
+    const containerProps =
+      props['aria-label'] || props['aria-labelledby'] || props.title
+        ? {
+            role: 'img',
```

Comment:
@tomi-msft is the new icon owner, and should be aware of this change and it's reason